src/platform/scene/scene.ts

```diff
@@ -0,0 +1,44 @@
+/** @alpha */
+export type Scene = {
+  main: string
+  scene: {
+    base: string
+    parcels: string[]
+  }
+  display?: {
+    title?: string
+    favicon?: string
+    description?: string
+    navmapThumbnail?: string
+  }
+  owner?: string
+  contact?: {
+    name?: string
+    email?: string
+    im?: string
+    url?: string
+  }
+  tags?: string[]
+  source?: {
+    origin?: string
+    projectId?: string
+  }
```

Comment:
Can it get some comments about what is it? What is projectId refering to? what is origin and source?

Reply:
This is metadata used by the Builder & Builder In-World. I'll let the dapps team explain how they are using it

Reply:
builder in world is using source property defined by builder dapp here https://github.com/decentraland/builder/blob/master/src/modules/deployment/types.ts#L67-L75
`origin` is used to establish if scene was deployed using builder ("builder") or builder in world ("builder-in-world")
 `projectId` is used to match the scene with a builder-server project
 `rotation` and `layout` are used to transform the deployed scene and keep compatibility with the builder-server project
 `isEmpty` is used to flag that the scene was "unpublished" which means that there was a builder in world scene pubished there and the owner opt to publish an empty-scene to replace it by clicking "unpublish" on the builder in world panel

Reply:
Yes, in the beginning the purpose of `source` was for analytics, by tracking the deployments' origin (CLI or Builder). In the end it ended up being metadata for the Builder, although this is not complete, the whole type would be like this (from the Builder repo):

```ts
type Coordinate = { x: number; y: number }

type Rotation = 'north' | 'east' | 'south' | 'west'

type Placement = { point: Coordinate; rotation: Rotation }

type Scene = {
  ...
  source?: {
    version?: number
    origin: string
    projectId: string
    point?: Placement['point']
    rotation?: Placement['rotation']
    layout?: Project['layout']
    isEmpty?: boolean
  }
}
```

Reply:
Ok, will update then